keyboards/acheron/arctic/keymaps/tsangan/keymap.c

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2012,2013 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+   [0] = LAYOUT_60_tsangan_hhkb(
```

Comment:
Given `default` also uses `LAYOUT_60_tsangan_hhkb`, i'm not sure how much value this keymap adds.

Reply:
Aside from this, it LGTM.  

I would prefer an answer here before it gets merged.

Reply:
Keymap removed in 46da2c094

Reply:
Keymap removed in 75a666c